libs/openFrameworks/utils/ofFileUtils.cpp

```diff
@@ -1925,17 +1925,18 @@ void ofSetDataPathRoot(const of::filesystem::path& newRoot){
 }
 
 //--------------------------------------------------
-// MARK: - near future
-//of::filesystem::path ofToDataPath(const of::filesystem::path & path, bool makeAbsolute){
 std::string ofToDataPath(const of::filesystem::path & path, bool makeAbsolute){
+	return ofToDataPathFS(path, makeAbsolute).string();
```

Comment:
.string() should be try-catched

Reply:
I've reverted this file as it is in master now. 
This change is still needed, I'll do in a separate PR finding all fuctions ending with FS in the name (which have a parallel string equivalent)